pkg/apis/core/v1/helper/helpers_test.go

```diff
@@ -301,6 +301,58 @@ func TestNodeSelectorRequirementsAsSelector(t *testing.T) {
 	}
 }
 
+func TestTopologySelectorRequirementsAsSelector(t *testing.T) {
+	mustParse := func(s string) labels.Selector {
+		out, e := labels.Parse(s)
+		if e != nil {
+			panic(e)
+		}
+		return out
+	}
+	tc := []struct {
+		in        []v1.TopologySelectorLabelRequirement
+		out       labels.Selector
+		expectErr bool
+	}{
+		{in: nil, out: labels.Nothing()},
+		{in: []v1.TopologySelectorLabelRequirement{}, out: labels.Nothing()},
+		{
+			in: []v1.TopologySelectorLabelRequirement{{
+				Key:    "foo",
+				Values: []string{"bar", "baz"},
+			}},
+			out: mustParse("foo in (baz,bar)"),
+		},
+		{
+			in: []v1.TopologySelectorLabelRequirement{{
+				Key:    "foo",
+				Values: []string{},
+			}},
+			expectErr: true,
+		},
+		{
+			in: []v1.TopologySelectorLabelRequirement{{
+				Key:    "/invalidkey",
+				Values: []string{"bar", "baz"},
+			}},
+			expectErr: true,
+		},
+	}
```

Comment:
Some more test cases:
* Multiple TopologySelectorLabelRequirement: success and fail
* Multiple Values

Reply:
what do you mean by "Multiple Values" here?